docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -7,41 +7,61 @@ description: Dagster supports data backfills for each partition or subsets of pa
 
 ## Overview
 
-Dagster supports data backfills for each partition or subsets of partitions. After defining a [partitioned job](/concepts/partitions-schedules-sensors/partitions), you can use backfills to submit runs for each partition in the set.
+Dagster supports data backfills for each partition or subsets of partitions. After defining a [partitioned asset or job](/concepts/partitions-schedules-sensors/partitions), you can use backfills to submit runs for each partition in the set.
 
 ---
 
 ## Launching Backfills
 
-### Using Dagit
+### Partitioned Asset Backfills
+
+You can open the backfill modal to launch backfills for a partitioned asset using the "Materialize" button on the [Asset detail page](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions#materializing-partitioned-assets).
+
+<Image
+  alt="backfills-launch-modal"
+  src="/images/concepts/assets/asset-backfill-partition-selection-modal.png"
+  width={3808}
+  height={2414}
+/>
+
+After selecting the partitions you want to backfill you can click on the toast notification to go to the Backfill details page. You can also reach this page via link on the backfills page.
+
+<Image
+  alt="backfills-launch-modal"
+  src="/images/concepts/assets/asset-backfill-details-page.png"
+  width={3808}
+  height={2414}
+/>
+
+### Partitioned Job Backfills
 
 You can launch and monitor backfills of a job using the [Partitions tab](/concepts/partitions-schedules-sensors/partitions#dagit-partitions-tab).
 
-To launch a backfill, click the "Launch backfill" button at the top center of the Partitions tab. This opens the "Launch backfill" modal, which lets you select the set of partitions to launch the backfill over. A run will be launched for each partition.
+To launch a backfill, click the "Materialize" button at the top right of the Partitions tab. This opens the "Launch backfill" modal, which lets you select the set of partitions to launch the backfill over. A run will be launched for each partition.
```

Comment:
I think the button is still "launch backfill" for the op-job case.

Reply:
Ah yeah, I was looking at a different tab